### DISCUSSION TITLE
Keeping student's names anonymous

### DISCUSSION THREAD

**Discussion by Sukhman-Singh** [sandboxnu/office-hours#617] [Ideas]
I noticed that recently the app stopped showing students' names to other students while waiting in queue. However, it still says the name when a TA is helping someone (like Helping _______ for 1 min). This isn't really a big issue, but if you wanted to keep the anonymity, you can have it say "Helping a student for 1 min" and make the actual name visible only to TAs and professors.

**Comment by dfarooq610** [comment]
Hey! We actually added this as an issue to tackle a couple of days ago! Thanks for the feedback! See #604
